lte/gateway/python/magma/pipelined/app/inout.py

```diff
@@ -163,12 +162,22 @@ def _install_default_middle_flows(self, dp):
                                              resubmit_table=next_tbl)
 
         if self._mtr_service_enabled:
-            match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
+            match = MagmaMatch(vlan_vid=(0x0000, 0x1000),
```

Comment:
make this tuple a const

Reply:
This is used in other pipelineD apps, I will take care of this as part of separate task.